_plugins/figures/iiif/manifest/index.js

```diff
@@ -58,13 +58,59 @@ module.exports = class Manifest {
     return this.createAnnotation({
       body: {
         items,
-        type: 'Choice',
+        type: 'Choice'
       },
       id: 'choices',
-      motivation: 'painting',
+      motivation: 'painting'
     })
   }
 
+  // TODO create annotations for annotation targets matching sequence item
+  get sequence() {
+    if (!this.figure.sequence) return
+    return this.figure.sequence
+      .map(({ items }) => {
+        // return this.createAnnotation({
+        //   body: {
+        //     items: items.map((item) => this.createAnnotationBody(item)),
+        //     type: 'Choice'
+        //   },
+        //   id: 'sequence-item',
+        //   motivation: 'painting'
+        // })
+
+        const canvasId = path.join(this.figure.canvasId, items[0].id)
+        return {
+          body: {
+            items: items.map(({ format, info, label, src, uri }) => {
+              const { ext } = path.parse(src)
+              return {
+                format,
+                height: this.figure.canvasHeight,
+                id: uri,
+                label: { en: [label] },
+                type: 'Image',
+                service: info && [
+                  {
+                    '@context': 'http://iiif.io/api/image/2/context.json',
+                    '@id': info,
+                    profile: 'level0',
+                    protocol: 'http://iiif.io/api/image'
+                  }
+                ],
```

Comment:
should probably separate this out into a class method now that it has multiple uses, somethin like `createImageService(info)`